livekit/src/room/participant/remote_participant.rs

```diff
@@ -353,6 +353,32 @@ impl RemoteParticipant {
                 }
             }
         });
+
+        publication.on_enabled_status_changed({
+            let rtc_engine = self.inner.rtc_engine.clone();
+            move |publication, enabled| {
+                let rtc_engine = rtc_engine.clone();
+                livekit_runtime::spawn(async move {
+                    let tsid: String = publication.sid().into();
+                    let TrackDimension(width, height) = publication.dimension();
+                    let update_track_settings = proto::UpdateTrackSettings {
+                        track_sids: vec![tsid.clone()],
+                        disabled: !enabled,
+                        width,
+                        height,
+                        quality: proto::VideoQuality::High.into(), // deprecated, but not marked in proto
+                        priority: 0,
+                        fps: 0,
+                    };
+
+                    let _ = rtc_engine
```

Comment:
I don't think we should ignore the error here?

Reply:
This is still not ideal. Returning a value from a tokio task is still ignoring the error.
I think what you can do is to log an error when not OK

Reply:
`send_request` actually returns `()`, so there's nothing to do here, and `let _` isn't needed. there's no error value returned that we can check for. other PR does have this issue though